jumanji/environments/routing/lbf/viewer.py

```diff
@@ -0,0 +1,301 @@
+        plt.close(self._name)
+
+    def _clear_display(self) -> None:
+        if jumanji.environments.is_colab():
+            import IPython.display
+
+            IPython.display.clear_output(True)
+
+    def _get_fig_ax(self) -> Tuple[plt.Figure, plt.Axes]:
+        recreate = not plt.fignum_exists(self._name)
+        fig = plt.figure(
+            self._name, figsize=constants._FIGURE_SIZE, facecolor=constants._GRID_COLOR
+        )
+        fig.subplots_adjust(left=0, bottom=0, right=1, top=1, wspace=0, hspace=0)
+
+        if recreate:
+            fig.tight_layout()
+            if not plt.isinteractive():
+                fig.show()
+            ax = fig.add_subplot(111)
+        else:
+            ax = fig.get_axes()[0]
+        return fig, ax
+
+    def _prepare_figure(self, ax: plt.Axes) -> None:
+        ax.set_xlim(0, self.width)
+        ax.set_ylim(0, self.height)
+        ax.patch.set_alpha(0.0)
+        ax.set_axis_off()
+
+        ax.set_aspect("equal", "box")
+        ax.set_xticks([])
+        ax.set_yticks([])
+        ax.set_frame_on(False)
+
+    def _draw_state(self, ax: plt.Axes, state: State) -> None:
+        self._draw_grid(ax)
+        self._draw_food(state.food_items, ax)
+        self._draw_agents(state.agents, ax)
+
+    def _draw_grid(self, ax: plt.Axes) -> None:
+        """Draw the grid."""
+        lines = [
+            [
+                (0, (self.grid_size + 1) * r + 1),
+                ((self.grid_size + 1) * self.cols, (self.grid_size + 1) * r + 1),
+            ]
+            for r in range(self.rows + 1)
+        ]
+        # HORIZONTAL LINES
+        lines.extend(
+            [
+                ((self.grid_size + 1) * c + 1, 0),
+                ((self.grid_size + 1) * c + 1, (self.grid_size + 1) * self.rows),
+            ]
+            for c in range(self.cols + 1)
+        )
+        lc = LineCollection(lines, colors=(constants._LINE_COLOR,))
+        ax.add_collection(lc)
+
+    def _display_human(self, fig: plt.Figure) -> None:
+        if plt.isinteractive():
+            # Required to update render when using Jupyter Notebook.
+            fig.canvas.draw()
+            if jumanji.environments.is_colab():
+                plt.show(self._name)
+        else:
+            # Required to update render when not using Jupyter Notebook.
+            fig.canvas.draw_idle()
+            fig.canvas.flush_events()
+
+    def _display_rgb_array(self, fig: plt.Figure) -> NDArray:
+        fig.canvas.draw()
+        return np.asarray(fig.canvas.buffer_rgba())
+
+    def _draw_agents(self, agents: Agent, ax: plt.Axes) -> None:
+        """Draw the agents on the grid."""
+        num_agents = len(agents.level)
+
+        for i in range(num_agents):
+            agent = tree_slice(agents, i)
+            cell_center = self._entity_position(agent)
+
+            # Read the image file
+            img = mpimg.imread(
+                os.path.join(os.path.dirname(__file__), "icons/agent.png")
+            )
```

Comment:
Can you copy how we do it [here](https://github.com/instadeepai/jumanji/pull/251/files) - the way the files are read is very important when it's installed via pip as it won't pick up the image files. Also could you store them in a folder called `imgs/` as is done in sokoban/tsp etc